app/styles/app/modules/request/form.scss

```diff
@@ -0,0 +1,93 @@
+.request-configs-form {
+  .config-type {
+    position: absolute;
+    top: 1px;
+    right: 0;
+    font-size: 10px;
+    z-index: 2;
+
+    * {
+      display: inline-block;
+      padding: 0 8px;
+      border: 1px solid transparent;
+      border-radius: 4px;
+    }
+
+    &.javascript .javascript,
+    &.yaml .yaml {
+      color: white;
+      background-color: $oxide-blue;
+      border-color: $oxide-blue;
+    }
+  }
+
+  .travis-form__field-component {
+    box-shadow: inset 0 0 1px 1px $dry-cement !important;
```

Comment:
Same here about use of `!important`

Reply:
i believe i've addressed this in https://github.com/travis-ci/travis-web/commit/5003d2b2e8ab86a2dbb2819bf4015c6e3255d755

Reply:
Nice, thanks!